idaes/generic_models/unit_models/__init__.py

```diff
@@ -16,3 +16,4 @@
 from .statejunction import StateJunction
 from .translator import Translator
 from .valve import ValveFunctionType, Valve, ValveData
+from .surrogate_model import SurrogateModel
```

Comment:
May as well import SurrogateModelData here too if you may want to subclass the surrogate block.  If not disregard.